lms/static/sass/base/_layouts.scss

```diff
@@ -0,0 +1,104 @@
+// base layout styles to support early responsive lms
+// may be discarded later once sass breakpoints are wired in
+
+body.view-incourse {
+  background-color: $body-bg;
+
+  .window-wrap {
+    min-width: 760px;
```

Comment:
Because of the scattered implementation of responsive, I decided to keep this scoped to this specific use and kept it hardcoded until we can make things more universal/variable-friendly.